src/components/Dot/index.tsx

```diff
@@ -11,8 +12,10 @@ const StyledDot = styled(Box, {
   border-radius: 50%;
   width: 10px;
   height: 10px;
-  background-color: ${({ theme: { colorsDeprecated }, color }) =>
-    colorsDeprecated[color as Color] ?? color};
+  background-color: ${({ theme, color }) =>
+    theme.colors[color as Color]?.backgroundStrong ??
+    theme.colorsDeprecated[color as ColorDeprecated] ??
```

Comment:
Why support both ? 🤔

Reply:
Probably because we are using a lot <Dot color="red | any color name">

Reply:
ok :)